lerna.json

```diff
@@ -1,15 +1,12 @@
 {
-  "commands": {
+  "command": {
     "publish": {
-      "ignore": [
+      "ignoreChanges": [
         "*.md",
         "*-spec.js"
       ]
     }
   },
-  "independent": true,
-  "lerna": "2.7.1",
-  "npmClient": "yarn",
```

Comment:
Now we'll just have Lerna use npm for its commands. Things just seem to work better that way.